### PR TITLE
[Hal 1457] - Console 'Failed to create security context' if there are no data-sources

### PR DESCRIPTION
**Pull request by adamecp**
JIRA upstream :  [https://issues.jboss.org/browse/HAL-1457](https://issues.jboss.org/browse/HAL-1457)

Description: In domain mode, if you add a xa-data-source and remove all of the data-sources, and reload server-one as requested. Then in the web console - Runtime / Server Group / main-server-group / server-one / Subsystems / Datasources / View gives error "Failed to create security context"

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/datasources/CreateSecurityContextTestCase.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        dsOps.createXADataSource(DATASOURCE_NEW_NAME, DATASOURCE_NEW_URL);
+        operations = new Operations(client);
+        administration = new Administration(client);
+        administration.reloadIfRequired();
+    }
+
+    @AfterClass
+    public static void tearDown() throws InterruptedException, TimeoutException, IOException, CommandFailedException {
+        client.apply(snapshotBackup.restore());
+        administration.reloadIfRequired();
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/datasources/CreateSecurityContextTestCase.java:66]: Please close `client` at the end.

**Review by zebrik — CHANGES_REQUESTED** [review]
Just one minor objection related to cleaning up otherwise LGTM.

**Review by adamecp — COMMENTED** [review]
I have done it.

**Review by zebrik — APPROVED** [review]
Seems OK now. Thank you!

**Pull request merged**